unity-renderer/Assets/Scripts/MainScripts/DCL/Configuration/Configuration.cs

```diff
@@ -216,18 +216,21 @@ public static UnityEngine.Vector2
 
     public static class PhysicsLayers
     {
-        public static int defaultLayer = LayerMask.NameToLayer("Default");
-        public static int onPointerEventLayer = LayerMask.NameToLayer("OnPointerEvent");
-        public static int characterLayer = LayerMask.NameToLayer("CharacterController");
-        public static int characterOnlyLayer = LayerMask.NameToLayer("CharacterOnly");
+        // Layers related to the components that can have physics. All models that have a withCollisions or isPointerBlocker will use them 
+        public static readonly int defaultLayer = LayerMask.NameToLayer("Default");
+        public static readonly int onPointerEventLayer = LayerMask.NameToLayer("OnPointerEvent");
+
+        // Layers related to the Avatar
+        public static readonly int characterLayer = LayerMask.NameToLayer("CharacterController");
+        public static readonly int collisionsWithoutPointerEvent = LayerMask.NameToLayer("CollisionsWithoutPointerEvent");
```

Comment:
name shouldn't be `CollisionsWithoutPointerEvent` since a layer represent a bit that will be on or off.
so name should indicate what does that bit is used for